Packs/SNDBOX/pack_metadata.json

```diff
@@ -2,9 +2,9 @@
     "name": "SNDBOX",
     "description": "SNDBOX as a service",
     "support": "xsoar",
-    "currentVersion": "1.0.5",
+    "currentVersion": "1.0.6",
     "author": "Cortex XSOAR",
-    "url": "https://www.paloaltonetworks.com/cortex",
+    "url": "https:\/\/www.paloaltonetworks.com\/cortex",
```

Comment:
you can remove this
```suggestion
```

Reply:
The build is failing because the "url" key is missing, I returned the URL back to normal ('https://www.paloaltonetworks.com/cortex').